src/background/sidebar-injector.js

```diff
@@ -48,24 +48,25 @@ function extractContentScriptResult(result) {
   }
 }
 
-/* The SidebarInjector is used to deploy and remove the Hypothesis sidebar
+/**
+ * The SidebarInjector is used to deploy and remove the Hypothesis sidebar
  * from tabs. It also deals with loading PDF documents into the PDF.js viewer
  * when applicable.
  *
- * chromeTabs - An instance of chrome.tabs.
- * dependencies - An object with additional helper methods.
- *   isAllowedFileSchemeAccess: A function that returns true if the user
+ * @param {chrome.tabs} chromeTabs
+ * @param {Object} services
+ * @param {(cb: (allowed: boolean) => void) => void} services.isAllowedFileSchemeAccess -
```

Comment:
Shouldn't this property be optional? There is a check in the code to test if exist and is a function.
```suggestion
 * @param {(cb: (allowed: boolean) => void) => void} [services.isAllowedFileSchemeAccess] -
```

Reply:
There is a check, but it throws an error if the parameters are missing. These are required parameters though. I'm going to remove the checks since we can now rely on TypeScript to verify that required parameters are passed.